.devcontainer/devcontainer.json

```diff
@@ -13,7 +13,11 @@
 		"PATH": "${containerEnv:PATH}:${containerEnv:FLUTTER_ROOT}/bin"
 	},
 	"onCreateCommand": "git clone --branch=stable https://github.com/flutter/flutter.git ${containerEnv:FLUTTER_ROOT} && ${containerEnv:FLUTTER_ROOT}/bin/flutter doctor",
+<<<<<<< HEAD
```

Comment:
Did you mean to leave these in?

Reply:
<sigh> I didn't even touch those files but figured I'd address it when I'm ready to land this. @parlough, can you resolve this?

Reply:
Fixed